server/src/main/java/org/apache/druid/server/security/AuthValidator.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server.security;
+
+import org.apache.druid.common.utils.IdUtils;
+
+/**
+ * Utility functions to validate the an authorizer.
+ */
+public class AuthValidator
+{
+  private static final String AUTHORIZER_NAME = "authorizerName";
+  private static final String AUTHENTICATOR_NAME = "authenticatorName";
+
+  /**
+   * Validates the provided authorizerName.
+   *
+   * @param authorizerName the name of the authorizer.
+   * @throws IllegalArgumentException on invalid authorizer names.
+   */
+  public void validateAuthorizerName(String authorizerName)
```

Comment:
Why are these methods not static? It doesn't seem to need to inject this class if they are static.

Reply:
It's easier to test non static methods when they are used in other classes